torch/distributed/elastic/agent/server/health_check_server.py

```diff
@@ -63,3 +244,35 @@ def create_healthcheck_server(
     creates health check server object
     """
     return HealthCheckServer(alive_callback, port, timeout)
+
+
+def create_gpu_healthcheck_server(
```

Comment:
can we expose a config option and cli arg via torchrun to enable this from the CLI?